.idea/dictionaries/alexf.xml

```diff
@@ -3,13 +3,18 @@
     <words>
       <w>blockchain</w>
       <w>calldata</w>
+      <w>calldatagascost</w>
```

Comment:
Why is this file indexed?

Reply:
So it can be checked out. I do look at the spelling errors in code and comments. Also, though the files are per-user, idea seems to take all dictionaries into account so everyone is invited to check in theirs.